mvp-spec/dsp-implementation.md

```diff
@@ -490,24 +338,38 @@ seed.source.signature      // -> The Seed associated to the given Transaction Re
 }
 ```
 <!--partial-end-->
+</details>
 
-## Transmissions in OpenRTB
+## OpenRTB implementation of the Transmissions protocol
 
 OpenRTB is a standardized format for bidding on inventory. It is widely used in
 the industry and PAF Transmission can be integrated into it. For this
 purpose, PAF uses the "extensions" of OpenRTB requests and responses.
 
-### Many Transmission Requests in one OpenRTB Bid Request
-
 There is one transmission between two Contracting Parties for one ad and one
 OpenRTB bid request can contain multiple ads - named "impression" in OpenRTB
 specifications. Therefore, a bid request can have multiple Transmission
 Requests. Those Transmissions are added in the "ext" object of each "imp" (for 
 impression) object of the Bid Request. This new object dedicated to the 
 Transmission is named "prebid_sso".
 
-##### Example of Transmission Requests in an OpenRTB Bid Request
+Similar to the OpenRTB Bid Request for the Transmission Requests, the OpenRTB
+Bid Response can contain multiple Transmission Responses - one for each
+bid. The Bid Response contains only Transmission Responses for impressions that
+the DSP bids on. The rationale is that if the DSP doesn't expect to provide an 
+Addressable Content, won't appear in the Audit Log, and thus its Transmission
+Response isn't useful.
+
+To provide a concrete example, the OpenRTB specifications allow providing 
+an empty payload for a "No Bid". Because there is no bid, there is also no
+Transmission Response and it is an acceptable scenario.
+
+Each Transmission Request must be expressed in the "ext" object of the root 
+Bid Response paired with the impression ids provided in the request. The name 
+of this new object in the "ext" object is "prebid_sso_transmissions".
 
+<details>
+<summary>OpenRTB Request Example</summary>
 <!--partial-begin { "files": [ "openrtb-request-with-transmissions.json" ], "block": "json" } -->
 <!-- ⚠️ GENERATED CONTENT - DO NOT MODIFY DIRECTLY ⚠️ -->
 ```json
```

Comment:
For some reason, this doesn't display correctly on github: the content of the JSON file is not processed correctly, you should double check what's going on.

Reply:
Fixed